coverity-scan.sh

```diff
@@ -1,38 +1,54 @@
 #!/usr/bin/env bash
 
+# To run this script you need to provide API token. This can be done either by:
+#  - Putting token in ".coverity-token" file
+#  - Assigning token value to COVERITY_SCAN_TOKEN environment variable
+# Additionally script can install coverity tool on your computer. To do this just set environment variable INSTALL_COVERITY to "true"
+
 cpus=$(grep -c ^processor </proc/cpuinfo)
 [ -z "${cpus}" ] && cpus=1
 
 token="${COVERITY_SCAN_TOKEN}"
 ([ -z "${token}" ] && [ -f .coverity-token ]) && token="$(<.coverity-token)"
-[ -z "${token}" ] && \
-	echo >&2 "Save the coverity token to .coverity-token or export it as COVERITY_SCAN_TOKEN." && \
+if [ -z "${token}" ]; then
+	echo >&2 "Save the coverity token to .coverity-token or export it as COVERITY_SCAN_TOKEN."
 	exit 1
-
-# echo >&2 "Coverity token: ${token}"
+fi
 
 covbuild="$(which cov-build 2>/dev/null || command -v cov-build 2>/dev/null)"
 ([ -z "${covbuild}" ] && [ -f .coverity-build ]) && covbuild="$(<.coverity-build)"
-[ -z "${covbuild}" ] && \
-	echo "Save command the full filename of cov-build in .coverity-build" && \
-	exit 1
-
-[ ! -x "${covbuild}" ] && \
-	echo "The command ${covbuild} is not executable. Save command the full filename of cov-build in .coverity-build" && \
+if [ -z "${covbuild}" ]; then
+	echo "Cannot find 'cov-build' binary in \$PATH."
+	if [ $INSTALL_COVERITY != "" ]; then
+		echo "Installing coverity..."
+		mkdir /tmp/coverity
+        	curl -SL --data "token=${token}&project=netdata%2Fnetdata" https://scan.coverity.com/download/linux64 > /tmp/coverity_tool.tar.gz
+        	tar -x-C /tmp/coverity/ -f /tmp/coverity_tool.tar.gz
+	        sudo mv /tmp/coverity/cov-analysis-linux64-2017.07 /opt/coverity
+        	export PATH=${PATH}:/opt/coverity/bin/
+        else
+		echo "Saving command the full filename of cov-build in .coverity-build"
```

Comment:
typo. It should say `Save the full filename of the cov-build command in .covertity-build`.